pkg/storage/backup/cbt.go

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
We are missing unit tests for this package

Reply:
Its in the last commit